{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason\/revokedByDevice"},"schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"modules":[{"name":"MobilePaymentsSDK"}],"externalID":"c:@M@MobilePaymentsSDKAPI@E@SQMPReaderConnectionFailureReason@SQMPReaderConnectionFailureReasonRevokedByDevice","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"revokedByDevice","kind":"identifier"}],"title":"ReaderConnectionFailureReason.revokedByDevice","roleHeading":"Case","symbolKind":"case"},"kind":"symbol","primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"revokedByDevice","kind":"identifier"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason"]]},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason\/revokedbydevice"]}],"abstract":[{"type":"text","text":"An error with the device occurred while attempting to establish a connection to Square."}],"references":{"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/ReaderConnectionFailureReason":{"role":"symbol","navigatorTitle":[{"text":"ReaderConnectionFailureReason","kind":"identifier"}],"identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason","url":"\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason","title":"ReaderConnectionFailureReason","abstract":[{"type":"text","text":"Indicates why a reader might have failed to connect to Square."}],"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ReaderConnectionFailureReason","kind":"identifier"}],"kind":"symbol"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI":{"abstract":[],"identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","url":"\/documentation\/mobilepaymentssdkapi","type":"topic","title":"MobilePaymentsSDK","kind":"symbol","role":"collection"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/ReaderConnectionFailureReason/revokedByDevice":{"url":"\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason\/revokedbydevice","title":"ReaderConnectionFailureReason.revokedByDevice","identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason\/revokedByDevice","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"revokedByDevice","kind":"identifier"}],"type":"topic","kind":"symbol","abstract":[{"type":"text","text":"An error with the device occurred while attempting to establish a connection to Square."}]}}}